src/braket/ocean_plugin/braket_sampler.py

```diff
@@ -273,16 +303,34 @@ def sample_qubo(self, Q: Dict[Tuple[int, int], int], **kwargs) -> SampleSet:
 
         Examples:
             This example submits a two-variable QUBO mapped directly to qubits
-            0 and 4 on a sampler
+            0 and 4 on a sampler on the D-Wave 2000Q device.
 
             >>> from braket.ocean_plugin import BraketSampler
-            >>> sampler = BraketSampler(s3_destination_folder, "device_arn_1")
+            >>> device_arn_1 = "arn:aws:braket:::device/qpu/d-wave/DW_2000Q_6"
+            >>> sampler = BraketSampler(s3_destination_folder, device_arn_1)
             >>> Q = {(0, 0): -1, (4, 4): -1, (0, 4): 2}
             >>> sampleset = sampler.sample_qubo(Q, postprocessingType="SAMPLING", shots=100)
             >>> for sample in sampleset.samples():
             ...    print(sample)
             ...
+            {0: 0, 4: 1}
             {0: 1, 4: 0}
+            {0: 0, 4: 0}
+            {0: 1, 4: 1}
```

Comment:
These two samples should be dropped from the example. They are not the minimum energy solutions and then the output from this example would match the Advantage one and the qubo_task one.

Reply:
Should the example be changed for this?